### ISSUE TITLE
Support customizing data displayed for TUI search

### ISSUE THREAD

**Issue by lazywei** [atuinsh/atuin#998]
I'm building `atuin` on HEAD,

This is working as intended
```
atuin search --format "{time} {command}"
```

But the following doesn't, it just ignore the specified format
```
atuin search -i --format "{time} {command}"
```

Is this expected?

**Comment by conradludgate** [comment]
Yes, that is expected. I've been considering changing it to properly respect what flags are supported under interactive or not, but I haven't managed to find a decent solution yet

**Comment by iloveitaly** [comment]
My use case here is I don't care about anything other than `{command}` in my results and would love the tui list to be nice and clean.
